server/client/ruby/candlepin_api.rb

```diff
@@ -1227,6 +1236,11 @@ def post_text(uri, data=nil)
     return response.body
   end
 
+  def post_text_return_json(uri, data=nil)
```

Comment:
The existing post_text only accepts text a s a return. I am returning json, which blew the old method up.